package.json

```diff
@@ -12,32 +12,17 @@
   "scripts": {
     "build": "npm run build:css",
     "watch": "npm run watch:css",
-    "build-dev": "npm run build-dev:css",
-    "watch-dev": "npm run watch-dev:css",
 
-    "build:css": "npm run build:less && npm run build:scssonly",
-    "watch:css": "npm run watch:less",
-    "build-dev:css": "npm run build-dev:less",
-    "watch-dev:css": "npm run watch-dev:less",
-
-    "build:less": "grunt less && npm run lessToSass",
-    "watch:less": "grunt watch:less",
-    "build-dev:less": "grunt lessdev && npm run lessToSass",
-    "watch-dev:less": "grunt watch:lessdev",
+    "build:css": "npm run build:scss",
```

Comment:
Since we're ripping a bunch of stuff out here, we should update the [npm wiki page](https://vufind.org/wiki/development:npm) accordingly. I've added a TODO checkbox for the task. I'm not sure if it's best to keep existing documentation but flag things as deprecated/removed, or to fully simplify it. I guess we should probably maintain some legacy documentation for a while, though my gut just wants to get rid of it quickly. :-)